pkgs/data/fonts/siji/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name = "siji-${date}";
+  date = "2016-05-14";
+
+  src = fetchgit {
```

Comment:
There is also another benefit of `fetchFromGitHub`. If you examine the code of `fetchgit`, it does not fetch the full repo, but only N last commits. So if the repo receives too many new commits, `fetchgit` will fail to checkout to the specified commit as it won't be fetched. (There is workaround, though.)